packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -186,6 +188,34 @@ export class TablePageObject<T extends TableRecord> {
         return anchor as Anchor;
     }
 
+    public getRenderedCellIcon(
+        rowIndex: number,
+        columnIndex: number
+    ): Icon | null {
+        const icon = this.getRenderedCellView(rowIndex, columnIndex).shadowRoot!
+            .firstElementChild;
+        if (!icon || !(icon instanceof Icon)) {
+            throw new Error(
+                `Icon not found at cell ${rowIndex},${columnIndex}`
+            );
+        }
+        return icon;
+    }
+
+    public getRenderedCellSpinner(
+        rowIndex: number,
+        columnIndex: number
+    ): Spinner | null {
```

Comment:
Same here. It doesn't look like this function can actually return `null` since it throws in that case.

```suggestion
    ): Spinner {
```

Reply:
Moot, since I removed it.